code/game/objects/items/storage/backpack.dm

```diff
@@ -889,6 +894,23 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 	fuel_type = "utnapthal"
 	has_gamemode_skin = TRUE
 
+/obj/item/storage/backpack/marine/engineerpack/flamethrower/afterattack(obj/target, mob/user, proximity)
+	if(!proximity) // this replaces and improves the get_dist(src,localbackpack) <= 1 checks used previously
```

Comment:
Comment has the old var you were using, but not a big deal

Reply:
removed